src/components/App.js

```diff
@@ -9,61 +9,69 @@ import yaml from 'js-yaml';
 let githubClient;
 
 export default class App extends Component {
-  constructor(props) {
```

Comment:
Because this is the style guide of this project and when you change the tabs in the file it is not possible to actually see the differences in the pull request.
About whether to use 2 or 4 spaces. I currently adhere to the popular AirBNB style-guide which requires to use 2 tabs. https://github.com/airbnb/javascript#whitespace--spaces